HTML-Assignment-3 (Design Layouts)/Design Layout Dashbaord.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
REason to use this tag?

Reply:
Basically used to render the website on Edge browser if the website is encountered with any version of IE browser so to not break compatibility. 

The tag itself is written via the boilerplate code in VS code for HTML documents. I wasn't sure if we would require it or not. So kept as it is.